src/AddressValidator.php

```diff
@@ -46,6 +46,11 @@ public function validate(Address $address): ValidatedAddress
         $validateAddressesResponse = $this->gateway->validateAddresses(
             new ValidateAddressesRequest([$address], $this->options)
         );
+        
+         if(isset($validateAddressesResponse->responseBody()['ValidateAddressesResponse']['GeneralError'])) {
+            $msg = 'BPOST ' . $validateAddressesResponse->responseBody()['ValidateAddressesResponse']['GeneralError']['ErrorSeverity'] . ' ERROR: ' . $validateAddressesResponse->responseBody()['ValidateAddressesResponse']['GeneralError']['ErrorCode'];
+            throw new \Exception($msg);
```

Comment:
Could you create a specific exception class for this? Similar to this one: https://github.com/spatie/bpost-address-webservice/blob/6066ee058aafd3fe17da782c9c3cf9aa05d7d711/src/AddressValidator.php#L58